domain/build.gradle

```diff
@@ -0,0 +1,77 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+apply from: '../config/jacoco.gradle'
+
+def globalConfig = rootProject.extensions.getByName("ext")
+android {
+    compileSdkVersion globalConfig.getAt("android_compile_sdk_version")
+    defaultConfig {
+        minSdkVersion globalConfig.getAt("android_min_sdk_version")
+        targetSdkVersion globalConfig.getAt("android_target_sdk_version")
+        versionCode globalConfig.getAt("android_version_code")
+        versionName globalConfig.getAt("android_version_name")
+    }
+
+    flavorDimensions "stage"
+    productFlavors {
+        production {
+            dimension "stage"
+        }
+
+        staging {
+            dimension "stage"
+        }
+
+        development {
+            dimension "dev"
+        }
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled true
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+        debug {
+            minifyEnabled false
+        }
+    }
+
+    sourceSets {
+        test {
+            resources.srcDirs += ['src/test/resources']
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    androidExtensions {
+        experimental = true
+    }
+}
+
+dependencies {
+    api project(':data')
+    implementation project(':common-rx')
```

Comment:
Hmmm, just wondering. I don't see any references to `project(':domain')` ? Is not it being used, or did I miss something 🙈  ?

Reply:
@Tuubz out of PRs scope https://github.com/nimblehq/android-templates/pull/48/files#diff-51a0b488f963eb0be6c6599bf5df497313877cf5bdff3950807373912ac1cdc9R124-R125

Reply:
yeah, currently we haven't applied use cases in main app yet, it will be coming on other PR 🙏